tracing-tower/Cargo.toml

```diff
@@ -18,7 +18,7 @@ http = { version = "0.1", optional = true }
 
 [dev-dependencies]
 bytes = "0.4"
-h2 = "=0.1.11"
+h2 = "0.1.11"
```

Comment:
Similarly, this equals constraint had to be relaxed for `tower-hyper` to build.

Reply:
not sure I follow what error you were getting?

Reply:
the equals constraint causes us to depend on _exactly_ version 0.1.11 of `hyper`, which depends on an old version of `h2`, which doesn't have methods that `tower-hyper` expects, IIRC